Sources/StytchCore/StytchClient/OTP/SytchClient+OTP.swift

```diff
@@ -14,22 +14,23 @@ public extension StytchClient {
         // sourcery: AsyncVariants, (NOTE: - must use /// doc comment styling)
         /// Wraps Stytch's OTP [sms/login_or_create](https://stytch.com/docs/api/log-in-or-create-user-by-sms), [whatsapp/login_or_create](https://stytch.com/docs/api/whatsapp-login-or-create), and [email/login_or_create](https://stytch.com/docs/api/log-in-or-create-user-by-email-otp) endpoints. Requests a one-time passcode for a user to log in or create an account depending on the presence and/or status current account.
         public func loginOrCreate(parameters: Parameters) async throws -> OTPResponse {
-            try await router.post(to: .loginOrCreate(parameters.deliveryMethod), parameters: parameters)
+            try await router.post(to: .loginOrCreate(parameters.deliveryMethod), parameters: parameters, useDFPPA: true)
```

Comment:
For `loginOrCreate` and `send` OTP methods, it's correct for SMS/WhatsApp, but Email is not DFPPA protected, so will need to switch on the delivery method

Reply:
Ok fixed!